plugins/easyimage/plugin.js

```diff
@@ -93,9 +90,9 @@
 			toolbar: 'easyimage,3'
 		} );
 
-		editor.balloonToolbars.create( {
+		editor._.easyImageToolbarContext = editor.balloonToolbars.create( {
```

Comment:
Not really, it has to be a per-editor member as there will be multiple instances of Balloon Toolbar context, each for every editor. If you'd put it in a plugin scoped variable, it would be overridden by a single, most recent context.